assets/wizards/subscriptionsOnboarding/index.js

```diff
@@ -224,7 +224,7 @@ class SubscriptionsOnboardingWizard extends Component {
 }
 
 render(
-	createElement( withWizard( SubscriptionsOnboardingWizard, [ 'woocommerce' ] ), {
```

Comment:
Not sure why the WooCommerce requirement was removed. I get `Unrecoverable error: The WooCommerce plugin is not installed and activated. Install and/or activate it to access this feature.` if I try accessing the wizard without having WC installed.

Reply:
This was almost certainly a mistake on my part, probably to test something. Sorry, and good catch! https://github.com/Automattic/newspack-plugin/pull/114/commits/033095342e5b98cbdf3066a6b545f4c05e858f88